CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Since it is a boilerplate project, there are technically no official (versioned) _releases_. Therefore, the `master` branch should always be stable and usable.
 
+## 2020-06-17
+
+- Add MixAudit vulnerability security scanning (#?)
```

Comment:
This is as a placeholder to link to this PR?

Reply:
Oh right, I usually edit this after creating the PR 😉